commodore/compile.py

```diff
@@ -67,8 +41,8 @@ def _regular_setup(config: Config, cluster_id):
     update_params(config.inventory, cluster)
 
     # Fetch components and config
-    _fetch_global_config(config, cluster)
-    _fetch_customer_config(config, cluster)
+    fetch_global_config(config, cluster)
+    fetch_customer_config(config, cluster)
```

Comment:
Might be worth to consider refactoring everything in `_regular_setup` up to this line into another method which can be used here and for `list_components`.